bin/ds

```diff
@@ -162,9 +180,6 @@ function build {
     source ~/.bashrc
   fi
 
-  echo 'Adding APC script at /apc.php...'
-  ln -s /srv/salt/php5/apc.php $WEB_PATH/apc.php
-
```

Comment:
cool, I'm trying to test it now, just a little confused by all the if statements